src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -851,7 +851,7 @@ def find_child_collection(parent, *args, **kwargs):
     return collection
 
 
-def check_connectivity(url='https://azure.microsoft.com', max_retries=5, timeout=1):
+def check_connectivity(url='https://raw.githubusercontent.com', max_retries=5, timeout=1):
```

Comment:
This function is only used by `_update_latest_from_github`, we'd better use `raw.githubusercontent.com` to check Internet connection.